src/utilities/view.ts

```diff
@@ -20,19 +19,17 @@ import {
  * Generates a default generic UI view.
  */
 export function generateDefaultView(
-  activityTypes: ActivityType[] = [],
   resourceTypes: ResourceType[] = [],
   externalEventTypes: ExternalEventType[] = [],
 ): View {
   const now = new Date().toISOString();
-  const types: string[] = activityTypes.map(({ name }) => name);
 
   const timeline = createTimeline([], { marginLeft: 250, marginRight: 30 });
   const timelines = [timeline];
 
   // Start with the activity row
   const activityLayer = createTimelineActivityLayer(timelines, {
-    filter: { activity: { static_types: types } },
+    filter: /* { activity: { static_types: types } } */ {},
```

Comment:
can this comment be removed?

Reply:
Does the `Layer` type need to be updated so that the `filter` no longer takes an `activity` specification or is this specific to only handle default view generation?

Reply:
Removed comment and I don't think `Layer` needs any updates? `Layer` requires `filter` but all sub-objects like `activity` are optional.

Reply:
Okay. I couldn't remember if that was still the method to add activity types to the layer or if it was moved to the advanced filters.

Reply:
Ah, there is a separate function to update the layer filters.